### PR TITLE
Feature: Add tasks

### LINKED ISSUE #39: 4a. Enable user to add Ingredient to Fridge

### PR DESCRIPTION
**Pull request by alvynben**
Fixes #39 .

Expected User Output:

```
                                                                        
   ad88              88           88                                    
  d8"                ""           88                             ,d     
  88                              88                             88     
MM88MMM  8b,dPPYba,  88   ,adPPYb,88   ,adPPYb,d8   ,adPPYba,  MM88MMM  
  88     88P'   "Y8  88  a8"    `Y88  a8"    `Y88  a8P_____88    88     
  88     88          88  8b       88  8b       88  8PP"""""""    88     
  88     88          88  "8a,   ,d88  "8a,   ,d88  "8b,   ,aa    88,    
  88     88          88   `"8bbdP"Y8   `"YbbdP"Y8   `"Ybbd8"'    "Y888  
                                       aa,    ,88                       
                                        "Y8bbdP"                        
Hello!
What would you like to do?
_____________________________________
add burger /2021-09-23
_____________________________________
You have successfully added:
    burger | 2021-09-23
```

### CODE REVIEW

File: src/main/java/seedu/commands/AddCommand.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+     */
+    @Override
+    public void execute(Ui ui, Parser parser, IngredientList ingredientList) {
+        Ingredient newIngredient = parser.parseIngredientForAdding(ui.getCurrentUserInput());
+        ingredientList.addIngredient(newIngredient);
+        ui.printReactionToAddingIngredient(newIngredient);
+    }
+
+
+
```
Comment (zonglun99) [inline src/main/java/seedu/commands/AddCommand.java:28]: Just a minor thing, can delete these spaces

**Review by zonglun99 — COMMENTED** [review]
Very minor changes, other than that it looks good

File: src/main/java/seedu/parser/Parser.java [hunk 2]
```diff
@@ -10,10 +15,30 @@ public Parser() {
...
+     * Returns a Ingredient based on user input.
+     * @param userInput The input from the user in this manner - "add burger /2021-09-23"
+     * @return An ingredient
      */
-    public void parseCommand(String userInput) {
-        String[] splitCommand = userInput.trim().split(" ", 2);
+    public Ingredient parseIngredientForAdding(String userInput) {
+        String[] splitUserInput = userInput.trim().split(" ");
+        String ingredientName = splitUserInput[1];
+        LocalDate expiryDate = LocalDate.parse(splitUserInput[2].replace("/",""));
```
Comment (BryanElmer) [inline src/main/java/seedu/parser/Parser.java:41]: Maybe can add a space after the comma

File: src/main/java/seedu/fridget/Fridget.java [hunk 3]
```diff
@@ -1,21 +1,37 @@
...
     public void run() {
         ui.printIntroduction();
-        String userInput = ui.readUserInput();
-        System.out.println("Hello " + userInput);
+
+        try {
+            String userInput = ui.readUserInput();
+            ui.printSeparatorLine();
+            Command c = parser.parseCommand(userInput);
+            c.execute(ui,parser,ingredientList);
```
Comment (BryanElmer) [inline src/main/java/seedu/fridget/Fridget.java:30]: Maybe can add spaces after the commas here to make the code consistent :)

**Review by BryanElmer — CHANGES_REQUESTED** [review]
Minor changes, other than that LGTM!

File: src/main/java/seedu/commands/Command.java [hunk 4]
```diff
@@ -13,7 +17,5 @@ public Command() {
      * Executes the command.
      * @throws FridgetException exception thrown when Command class is executed.
```
[the remark this answers is not in the document]
Comment (alvynben) [reply src/main/java/seedu/commands/Command.java]: Done!

File: src/main/java/seedu/commands/AddCommand.java (hunk 1 above)
Comment (alvynben) [reply src/main/java/seedu/commands/AddCommand.java]: Done!

File: src/main/java/seedu/ui/Ui.java [hunk 5]
```diff
@@ -29,14 +34,40 @@ public void printIntroduction() {
...
+        System.out.println(SEPARATOR_LINE);
+    }
+
+    /**
+     * Prints a reaction to user successfully adding an ingredient.
+     * @param ingredient The ingredient the user has added.
+     */
+    public void printReactionToAddingIngredient(Ingredient ingredient) {
+        System.out.println("You have successfully added:");
+        System.out.println("    " + ingredient);
```
[the remark this answers is not in the document]
Comment (alvynben) [reply src/main/java/seedu/ui/Ui.java]: Done!

File: src/main/java/seedu/parser/Parser.java (hunk 2 above)
Comment (alvynben) [reply src/main/java/seedu/parser/Parser.java]: Done!

**Review by uosjapuelks — APPROVED** [review]
Looks good

**Review by zonglun99 — APPROVED** [review]

**Review by BryanElmer — APPROVED** [review]
Looks good!

**Pull request merged**